github-runner-manager/src/github_runner_manager/metrics/reconcile.py

```diff
@@ -33,3 +33,23 @@
     documentation="Total number of runners cleaned up",
     labelnames=[labels.FLAVOR],
 )
+DELETED_RUNNERS_TOTAL = Counter(
+    name="deleted_runners_total",
+    documentation="The number of runners deleted during reconciliation.",
```

Comment:
Is the concept of runner vs vm documented somewhere? The first question which came to my mind was if this includes vm deletion. We could be more precise here 

```suggestion
    documentation="The number of removed runners from GitHub during reconciliation.",
```